west.yml

```diff
@@ -157,7 +157,7 @@ manifest:
       groups:
         - hal
     - name: hal_espressif
-      revision: 8c2ae3b0017cadac3de1f57592e879a7bb6ef75d
+      revision: b3072f7c305b6bd789b00db8a0428af0bf18ea6f
```

Comment:
Please, update this to latest `e589b872d92d8515ce512b2d34f255d0bccb0e0f` as there is another PR (mcuboot sync) using that revision.